### ISSUE TITLE
Scale or Rotate RigidBody should be locked, not just alerted

### ISSUE THREAD

**Issue by rogeriodec** [godotengine/godot#26890]
**Godot version:**
3.1 RC1

**OS/device including version:**
Windows 10 x64

**Issue description:**
My main purpose was to create a 3D board that would interact with the environment.
In this way I created the following structure:
![](https://i.imgur.com/OKfKvEY.png)

Where `Tala` is a RigidBody. And to prevent `Tala` from falling, I switched 'Mode' to 'Kinematic':
![](https://i.imgur.com/fuAioMd.png)

Since RigidBody has two children (MeshInstance and CollisionShape), I have manually resized only RigidBody, so MeshInstance and CollisionShape should inherit this RigidBody dimension automatically (CollisionShape and MeshInstance remain untouchable):
![](https://i.imgur.com/cJnnG92.png)

So far so good. The camera preview image within the editor and the resulting image rendering match:

![](https://i.imgur.com/0etMrZj.png)

Result (F6):
![](https://i.imgur.com/8t4Y4qz.png)

Now I want to **rotate RigidBody 20 degrees**, so I just enter this line of code:
```
extends Spatial
func _ready():
	$Tala.rotate_z(deg2rad(20))
```
Then, pressing F6 I get this distorted image:
👎
![](https://i.imgur.com/PLVNsbr.png)

After several hours of testing and lots of talk through the DiscordApp, I was alerted that a **RigidBody NEVER could be scaled or rotated**.

At the same time, that **yellow alert triangle next to RigidBody did not appear**, as you can see in the first image...

And one day later, I talked more into Discord and someone asked me to change RigidBody (parent) to **KinematicBody**.
I did this and ... it worked!
I did not change anything in the code, nor in Transform, I just did the "Change Type" for KinematicBody ...
And this is the result (which was expected), but that did not work as long as the parent was RigidBody:
👍
![](https://i.imgur.com/VHYU0vm.png) 

---

I made sure to detail this well, because I think this is a BUG in RigidBody and not a feature.
It does not make sense to just warn the user that something will give problem.
If this is known to cause problems, this must be LOCKED to the end user.
This is an assumption of good systems engineering.

Still, I think this RigidBody limitation is a BUG.
In short, either fix this bug or block access to resizing and rotating a RigidBody.

I, as a beginner, took a lot to learn how to overcome this limitation of Godot.

I hope newbie users do not suffer that much.

---

**Steps to reproduce:**

1. Open the attached project.
2. Open `Tala.tscn`
3. Press F6 and you will see the **distorced** result
4. Change the current RigidBody `Tala` to KinematicBody.
5. Press F6 and you will see the **correct** result.

**Minimal reproduction project:**
[esteira 3.1.zip](https://github.com/godotengine/godot/files/2950221/esteira.3.1.zip)

**Comment by eon-s** [comment]
RigidBodies can be rotated (at runtime only in a way it do not interfere with the physics engine), the scale is the only problem here.

The behavior requested here is the same used on 2D and also Camera node do not allow scaling, so it may be just matter of doing the same with this node.

**Comment by akien-mga** [comment]
Duplicate of #5734
